test/deterministicmnlist/QuorumEntry.js

```diff
@@ -97,34 +97,42 @@ describe('QuorumEntry', function () {
   describe('quorum signatures', function () {
     it('Should verify a threshold signature', function () {
       var entry = new QuorumEntry(quorumEntryJSON);
-      var res = entry.isValidQuorumSig();
-      expect(res).to.be.true;
+      entry.isValidQuorumSig()
+        .then((res) => {
+          expect(res).to.be.true;
+        });
     });
     it('Should verify an aggregated member signature', function () {
       var mnList = new SimplifiedMNList(SMNListFixture.getFirstDiff());
       mnList.applyDiff(SMNListFixture.getSecondDiff());
       mnList.applyDiff(SMNListFixture.getQuorumHashDiff());
       var entry = new QuorumEntry(quorumEntryJSON);
-      var res = entry.isValidMemberSig(mnList);
-      expect(res).to.be.true;
+      entry.isValidMemberSig(mnList)
```

Comment:
If the promise won't be resolved then we just miss this assert. We need to use `chai-as-promised` here and make sure that promise is resolved.